crawler/file_handler.py

```
@@ -124,6 +247,88 @@ def extract_text(path: Path, ext: str) -> tuple[str, str]:
+                "ext": ext,
+                "file_key": filename,
+                "local_path": f"files/{filename}",
+                "mime_type": f"image/{img_type}",
+                "file_size": file_size,
+                "checksum": checksum,
+                "extracted_text": "",
+                "download_ok": True,
+                "parser": "none",
+                "parse_ok": False,
+            })
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)